src/main/scala/xiangshan/mem/lsqueue/LoadQueueReplay.scala

```diff
@@ -389,21 +438,33 @@ class LoadQueueReplay(implicit p: Parameters) extends XSModule
   def replayCanFire(i: Int) = coldCounter(i) >= 0.U && coldCounter(i) < ColdDownThreshold
   def coldDownNow(i: Int) = coldCounter(i) >= ColdDownThreshold
 
+  val s1_balanceOldestSelExt = (0 until LoadPipelineWidth).map(i => {
+    val wrapper = Wire(Valid(new BalanceEntry))
+    wrapper.valid := s1_oldestSel(i).valid
+    wrapper.bits.balance := cause(s1_oldestSel(i).bits)(LoadReplayCauses.bankConflict)
+    wrapper.bits.index := s1_oldestSel(i).bits
+    wrapper.bits.port := i.U
+    wrapper
+  })
+  val s1_balanceOldestSel = balanceReOrder(s1_balanceOldestSelExt)
```

Comment:
- remove bank conflict block. In old design, when multiple replay ports have only one bank conflict, the current replay cycle will only replay this bank conflicting instruction, blocking other ports, and the next replay cycle will replay the instruction of other ports. In latest design, LoadQueueReplay will exchange replay ports, for example, instruction **_a_** is replayed by tlb miss from port 0, instruction **_b_** is replayed by bank conflict from port 1. In old design, LoadQueueReplay will replay instruction **_b_** first, then replay instruction **_a_**. In latest design, instruction **_a_** is replayed  from port 1, instruction **_b_** is replayed  from port 0.